lib/emerald/nodes/list_base.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'treetop'
+require_relative 'node'
+
+# Base rule for lists of images, metas, styles, and scripts
+class ListBase < Node
+  def to_html(context)
+    elements[4].elements.map do |e|
+      attrs = e.elements[0].elements.map do |j|
+        "#{j.elements[0].text_value}='#{j.elements[2].to_html(context)}'"
```

Comment:
http://stackoverflow.com/questions/4015345/how-to-properly-escape-quotes-inside-html-attributes wow, TIL you need to encode special characters in attributes too. I'd make a test for this, our literal `to_html` may already do this